packages/deposit-address/src/get-deposit-address/isomorphic.ts

```diff
@@ -32,3 +120,66 @@ export async function getDepositAddress(
 
   return getDepositAddressFromAxelarNetwork(params, chainConfigs, dependencies);
 }
+
+export async function getNativeWrapDepositAddress(
+  params: DepositNativeWrapOptions,
+  dependencies: GetDepositServiceDependencies
+) {
+  const chainConfigs = await dependencies.configClient.getChainConfigs(
+    params.environment
+  );
+
+  validateChainIds([params.sourceChain, params.destinationChain], chainConfigs);
```

Comment:
with all these validations, can we externalize this to a helper function?

Reply:
yeah, this function is already exported here https://github.com/axelarnetwork/axelarjs/blob/833eff620eec627254213e85c4e7489403ab0a15/packages/deposit-address/src/get-deposit-address/helpers/validation.ts#L30

Reply:
these functions are already put under [helpers/validation.ts](https://github.com/axelarnetwork/axelarjs/blob/833eff620eec627254213e85c4e7489403ab0a15/packages/deposit-address/src/get-deposit-address/helpers/validation.ts#L30)

or did you mean putting all of these validations into a one big helper function?

```ts
  validateChainIds([params.sourceChain, params.destinationChain], chainConfigs);
  validateAsset(
    [params.sourceChain, params.destinationChain],
    params.asset,
    chainConfigs
  );
  validateAddress(
    params.destinationAddress,
    chainConfigs.chains[params.destinationChain.toLowerCase()] as ChainConfig
  );
 ```